bufferserver/src/test/java/com/datatorrent/bufferserver/server/ServerTest.java

```diff
@@ -70,7 +70,7 @@ public static void setupServerAndClients() throws Exception
     eventloopServer.start();
     eventloopClient.start();
 
-    instance = new Server(0, 4096,8);
+    instance = new Server(0, 4096, 8);
```

Comment:
IMO, we should either enforce space after comma in codestyle check (it is not supported by the version of checkstyle Apex uses) or avoid format changes.